### PR TITLE
Adding a stats page

### PR DESCRIPTION
**Pull request by shawnthompson**
@netlify /en/pages-to-review/

### CODE REVIEW

File: .github/workflows/update-analytics.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Update Analytics Data
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Runs every day at midnight UTC
```
Comment (duboisp) [inline .github/workflows/update-analytics.yml:5]: Every day? would every week or every 2 weeks be sufficient? Or one a month considering that page is only showing monthly data?

**Review by duboisp — COMMENTED** [review]
Just have one question regarding the timing of the CRON job.

The rest look good. Great work.

**Comment by shawnthompson** [comment]
@JulieBB23 Do we want the analytics to be part of the "About the Digital Accessibility Toolkit" section on the homepage?

image

**Comment by JulieBB23** [comment]
> @JulieBB23 Do we want the analytics to be part of the "About the Digital Accessibility Toolkit" section on the homepage?
> 
> image

Yes, i think it is a good idea to share it.
Approved :)

**Comment by shawnthompson** [comment]
Any objects of this going live:

- [Website usage analytics](https://deploy-preview-644--a11ycanada.netlify.app/en/website-usage-analytics/)
- [Analytique de l’utilisation du site Web](https://deploy-preview-644--a11ycanada.netlify.app/fr/analytique-de-lutilisation-du-site-web/)

**Pull request merged**